prowler/providers/aws/services/iam/iam_user_accesskey_unused/iam_user_accesskey_unused.py

```diff
@@ -3,40 +3,12 @@
 from prowler.lib.check.models import Check, Check_Report_AWS
 from prowler.providers.aws.services.iam.iam_client import iam_client
 
-maximum_expiration_days = 45
+maximum_expiration_days = iam_client.audit_config.get("max_unused_access_keys_days")
```

Comment:
Putting this line there makes the tests to fail, you have to move it under line 10.

And set a default value with the same value in `config.yaml`. 

https://docs.prowler.cloud/en/latest/developer-guide/checks/#using-the-audit-config